modules/serverless-ossm-secret-filtering-net-istio.adoc

```diff
@@ -4,13 +4,15 @@
 
 :_content-type: PROCEDURE
 [id="serverless-ossm-secret-filtering-net-istio_{context}"]
-= Improving net-istio memory usage by using secret filtering for {SMProductShortName}
+= Improving net-istio and net-kourier memory usage by using secret filtering for {SMProductShortName}
 
-By default, the link:https://aly.arriqaaq.com/kubernetes-informers/[informers] implementation for the Kubernetes `client-go` library fetches all resources of a particular type. This can lead to a substantial overhead when many resources are available, which can cause the Knative `net-istio` ingress controller to fail on large clusters due to memory leaking. However, a filtering mechanism is available for the Knative `net-istio` ingress controller, which enables the controller to only fetch Knative related secrets. You can enable this mechanism by adding an annotation to the `KnativeServing` custom resource (CR).
+By default, the link:https://aly.arriqaaq.com/kubernetes-informers/[informers] implementation for the Kubernetes `client-go` library fetches all resources of a particular type. This can lead to a substantial overhead when many resources are available, which can cause the Knative `net-istio` and `net-kourier` ingress controllers to fail on large clusters due to memory leaking. However, a filtering mechanism is available for the Knative `net-istio` and `net-kourier` ingress controllers, which enable the controllers to only fetch Knative related secrets. You can enable this mechanism by adding an annotation to the `KnativeServing` custom resource (CR).
```

Comment:
```suggestion
By default, the link:https://aly.arriqaaq.com/kubernetes-informers/[informers] implementation for the Kubernetes `client-go` library fetches all resources of a particular type. This can lead to a substantial overhead when many resources are available, which can cause the Knative `net-istio` and `net-kourier` ingress controllers to fail on large clusters due to memory leaking. However, a filtering mechanism is available for the Knative `net-istio` and `net-kourier` ingress controllers, which enable the controllers to only fetch Knative related secrets.
```

Reply:
The point is that now this mechanism is enabled by default.

Reply:
True that! I have removed it.